docs/docs/data-source/5-null-and-undefined-handling.md

```diff
@@ -1,6 +1,14 @@
 # Handling Null and Undefined Values in Where Conditions
 
-TypeORM provides fine-grained control over how `null` and `undefined` values are handled in where conditions through the `invalidWhereValuesBehavior` configuration option in your data source options. This applies to all operations that support where clauses, including find operations, query builders, and repository methods.
+In 'WHERE' conditions the values `null` and `undefined` are not strictly valid values in TypeORM.
+
+Passing a known `null` value is disallowed by TypeScript (when you've enabled `strictNullChecks` in tsconfig.json) at compile time. But the default behavior is for `null` values encountered at runtime to be ignored. Similarly, `undefined` values are allowed by TypeScript and ignored at runtime.
```

Comment:
Shall we remove the "TypeScript Considerations" section at the bottom of this doc, now that this is explained here?

Reply:
Yeah, definitely, good point 🙏